experimental/PropertyDDS/packages/property-common/src/guidUtils.ts

```diff
@@ -10,7 +10,7 @@
  */
 
 import base64js from "base64-js";
-import { generateRandomUInt32Array } from "../platform-dependent";
+import { v4 as uuid } from "uuid";
```

Comment:
@ChumpChief has [AB#7451](https://dev.azure.com/fluidframework/internal/_workitems/edit/7451): "Remove @fluid-experimental/platform-dependent package" (which I think is meant to say "Remove @fluid-internal/platform-dependent").
There are comments that suggest it wouldn't be this simple. Is this change viable?